server/remote_cache/scorecard/scorecard_test.go

```diff
@@ -284,7 +284,7 @@ type fakeBlobStore struct {
 func (bs *fakeBlobStore) ReadBlob(ctx context.Context, name string) ([]byte, error) {
 	tokens := strings.Split(name, "/")
 	if attempt := tokens[1]; attempt == "1" {
-		return proto.Marshal(bs.ScoreCard)
```

Comment:
ahahaha please tell me this doesn't actually matter

Reply:
It's test no big deal. but since i already know ScoreCard is an anomaly; so..